src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -509,31 +465,54 @@ public void doGet(HttpServletRequest request,HttpServletResponse response)
 		}
 	}
 
-	public void doPost(HttpServletRequest request,HttpServletResponse response){
-		M_log.debug("doPOST: Called");
-		try {
-			//determine if this is an LTI Call or a browser call?
-			if(request.getParameterMap().containsKey("oauth_consumer_key")){
-				processLti(request, response);
-				return;
-			}
-			canvasRestApiCall(request, response);
-		}catch(Exception e) {
-			M_log.error("POST request has some exceptions",e);
-		}
-	}
-
-	private void processLti(HttpServletRequest request,
-			HttpServletResponse response) throws IOException {
-		for (Object e : request.getParameterMap().entrySet()) {
-			Map.Entry<String, String[]> entry = (Map.Entry<String, String[]>) e;
-			String name = entry.getKey();
-			if(M_log.isDebugEnabled()){
-				for (String value : entry.getValue()) {
-					M_log.debug(name + " = " + value);
-				}
-			}
-		}
+    public void doPost(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
`doPost` Is actually the servlet function i can't change it. I am sure you know that, but i am trying to understand what's your point here